### PR TITLE
chore: add note about permissions required and the userName attribute in the SCIM docs

### PR DESCRIPTION
**Pull request by rholshausen**
Added note about permissions required
Also added note about the importance of having the userName set, otherwise the integration will not work correctly.

### CODE REVIEW

File: website/docs/docs/scim/main.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ PactFlow users and teams can be managed via a SCIM 2.0 compatible API. For detai
 * `/scim/User` - Manage PactFlow users
 * `/scim/Groups` - Manage PactFlow teams
 
+## Authentication
+
+The SCIM API endpoints require a Pactflow bearer token. It is recommended to use a 
+[PactFlow Service Account token](/docs/user-interface/settings/users#system-accounts) for this. This service account is
```
Comment (mefellows) [inline website/docs/docs/scim/main.md:26]: We usually refer to these as "System Accounts".

File: website/docs/docs/scim/okta.md [hunk 2]
```diff
@@ -33,10 +33,12 @@ Select the *"Integration"* item on the left, and then setup the SCIM connection.
 
 1. SCIM version must be set to 2.0.
 2. Base URL is your PactFlow instance domain with `/scim/` appended. **Replace &lt;instance&gt; with your actual instance name.**
-3. Set the unique identifier to the Okta `userName` attribute.
+3. Set the unique identifier to the Okta `userName` attribute. **It is important that this value is the same value that is passed in via the SAML authentication call.** 
 4. Depending on the direction you want the resources to flow, select the appropriate checkboxes. You want to at least push new users, new groups and profile updates to PactFlow.
 5. Authentication mode must be set to HTTP Header
-6. Set the Authorization to Bearer and set it to your PactFlow API token. You should use a [PactFlow Service Account token](/docs/user-interface/settings/users#system-accounts) for this.  
+6. Set the Authorization to Bearer and set it to your PactFlow API token. You should use a [PactFlow Service Account token](/docs/user-interface/settings/users#system-accounts) for this.
+   This service account is going to require the `user:manage:*` and `team:manage:*` permissions to be able to create or update those resources.
```
Comment (mefellows) [inline website/docs/docs/scim/okta.md:40]:
```suggestion
   This system account is going to require the `user:manage:*` and `team:manage:*` permissions to be able to create or update those resources.
```

File: website/docs/docs/scim/main.md (hunk 1 above)
Comment (mefellows) [inline website/docs/docs/scim/main.md:26]:
```suggestion
[PactFlow System Account token](/docs/user-interface/settings/users#system-accounts) for this. This system account is
```

**Review by mefellows — APPROVED** [review]
Thank you, just a few minor comments.

File: website/docs/docs/scim/main.md (hunk 1 above)
Comment (rholshausen) [reply website/docs/docs/scim/main.md:26]: Ah, yes, we do, don't we

**Pull request merged**